moveit2_tutorials.repos

```diff
@@ -1,12 +1,8 @@
 repositories:
-  moveit2:
-    type: git
-    url: https://github.com/ros-planning/moveit2
-    version: main
```

Comment:
Remove this because this is redundant for CI, instead I added it to instructions.